libparsec/crates/platform_mountpoint/src/unix/filesystem.rs

```diff
@@ -1121,7 +1113,23 @@ impl fuser::Filesystem for Filesystem {
 
         // TODO: support atime/utime change ?
 
-        reply.manual().error(libc::ENOSYS);
+        // Nothing to set, just return the file attr
+
+        let path = {
+            let inodes_guard = self.inodes.lock().expect("mutex is poisoned");
+            inodes_guard.get_path_or_panic(ino)
+        };
+
+        let ops = self.ops.clone();
+
+        self.tokio_handle.spawn(async move {
+            let res = getattr_from_path(&ops, path, ino, uid, gid).await;
+
+            match res {
+                Ok(attr) => reply.manual().attr(&TTL, &attr),
+                Err(errno) => reply.manual().error(errno),
+            }
+        });
```

Comment:
Is this code what fixes both copy/paste and save issues? In any case, we should add a comment explaining briefly why the changes below are needed. 

We (you) had a hard time investigating and figuring out what was going on. Let's track that here as a comment.

Reply:
There is a comment just above

Reply:
https://github.com/Scille/parsec-cloud/pull/9351#discussion_r1918227957